pages/user/[id].vue

```diff
@@ -259,6 +300,75 @@ const auth = await useAuth()
 const cosmetics = useCosmetics()
 const tags = useTags()
 
+const vintl = useVIntl()
+const { formatMessage, formats: fmt } = vintl
+
+const formatRelativeTime = useRelativeTime()
+
+const messages = defineMessages({
+  profileDownloadsStats: {
+    id: 'profile.stats.downloads',
+    defaultMessage: '<stat>{count}</stat> downloads}',
+  },
+  profileProjectsFollowersStats: {
+    id: 'profile.stats.projects-followers',
+    defaultMessage: '<stat>{count}</stat> followers of projects',
+  },
+  profileJoinedAt: {
+    id: 'profile.joined-at',
+    defaultMessage: 'Joined {ago}',
+  },
+  profileJoinedAtTooltip: {
+    id: 'profile.joined-at.tooltip',
+    defaultMessage: '{date} at {time}',
```

Comment:
You can format the dates right in the string, see [`date` type section of Message Syntax guide](https://formatjs.io/docs/core-concepts/icu-syntax#date-type).